Makefile

```diff
@@ -1,6 +1,13 @@
 
 default: build
 
+QRI_VERSION?="0.9.14-dev"
+
+BUILD_FLAGS?=CGO_ENABLED=0
+PKG=$(shell go list ./version)
+GOLANG_VERSION=$(shell go version | awk '{print $$3}')
+GOVVV_FLAGS=$(shell govvv -flags -pkg $(PKG) -version $(QRI_VERSION))
+
 require-goversion:
```

Comment:
Would recommend also adding:

```
require-govvv:
	go install github.com/ahmetb/govvv
```

and adding that dependency to `build` and `install`.